### PR TITLE
avocado-vt: add coloured result in the test status info

### PR DESCRIPTION
**Pull request by hereischen**
add coloured result in the test status info, so the results will be more noticeable.
Signed-off-by: Haotong Chen <hachen@redhat.com>

ID: 1414639

### CODE REVIEW

**Comment by CongLi** [comment]
@hereischen 

Please add bug id for this pr.

**Comment by hereischen** [comment]
@CongLi , added bug id.

File: scripts/scan_results.py [hunk 1]
```diff
@@ -8,10 +8,23 @@
...
 import argparse
+
 from avocado.core import data_dir
 from dateutil import parser as dateparser
 
 
+def colour_result(result):
+    """Colour result in the test status info"""
+    possible_results = ('PASS', 'ERROR', 'FAIL')
+    if result == possible_results[0]:
```
Comment (xutian) [inline scripts/scan_results.py]: How about?
```
colours_map = {"PASS": "\033[92m PASS \033[00m"，
                         “ERROR”： "\033[93m ERROR \033[00m"，
                         “FAIL”: "\033[91m FAIL \033[00m"}
return  colours_map.get(result)  or result
```

Comment (hereischen) [inline scripts/scan_results.py]: That's better, will update it. Thanks

Comment (hereischen) [inline scripts/scan_results.py]: That's better, will update it. Thanks

**Comment by hereischen** [comment]
updated @xutian , please check.Many thanks

**Comment by xutian** [comment]
looks OK to me.

**Comment by pingl2016** [comment]
Acked-by: Ping Li <pingl@redhat.com>

**Pull request merged**